src/test/e2e/api/client/metrics.access.e2e.test.ts

```diff
@@ -33,6 +33,12 @@ test('should enrich metrics with environment from api-token', async () => {
         project: '*',
     });
 
+    const featureName = Object.keys(metricsExample.bucket.toggles)[0];
+    // @ts-expect-error
```

Comment:
Same comment about ts-expect-error

Reply:
Addressed in https://github.com/Unleash/unleash/pull/9957/commits/88d71ec1c4b4490acd23d647dfe6bd1d751bb51b